Compilation error on recent Fedora system

I tried to compile according to the instructions, but no luck:

$ make
cc -c -MMD -Os -I. -Ilittlefs -std=c99 -Wall -pedantic -D_FILE_OFFSET_BITS=64 -D_XOPEN_SOURCE=700 lfs_fuse_bd.c -o lfs_fuse_bd.o
lfs_fuse_bd.c:15:10: fatal error: stropts.h: No such file or directory
 #include <stropts.h>
          ^~~~~~~~~~~
compilation terminated.
make: *** [Makefile:51: lfs_fuse_bd.o] Error 1


Apparently stropts.h has not been distributed in Fedora for about a decade, so probably it should be removed.

--- a/lfs_fuse_bd.c
+++ b/lfs_fuse_bd.c
@@ -12,7 +12,7 @@
 #include <stdint.h>
 #include <assert.h>
 #if !defined(__FreeBSD__)
-#include <stropts.h>
+#include <sys/ioctl.h>
 #include <linux/fs.h>
 #elif defined(__FreeBSD__)
 #define BLKSSZGET DIOCGSECTORSIZE

